source/developer/slash-commands.rst

```diff
@@ -171,9 +171,9 @@ Tips and Best Practices
 Share Your Integration
```

Comment:
@aaronrothschild, the link to "getting started" includes a link to the "share your project" page which gives a 404. Wondering if we should direct contributors to rather post in a channel to share their app? Or linking to a more relevant page? 

It seems like this may work better if it links to the steps required to list an app? Maybe the contribution checklist: https://developers.mattermost.com/contribute/getting-started/contribution-checklist/?

Reply:
Yeah this is the problem all along

Reply:
@justinegeffen yes, that works fine.